RedHat-UX/red-hat-design-system
# docs(accordion): add information about accent slot

## What I did

1. Updated the Style and Guideline subpages with info about accent slot


## Testing Instructions

Check the following sections for new info:
- Style: [Anatomy](https://deploy-preview-1564--red-hat-design-system.netlify.app/elements/accordion/style/#anatomy)
- Style: [Space](https://deploy-preview-1564--red-hat-design-system.netlify.app/elements/accordion/style/#space)
- Style: [Configuration](https://deploy-preview-1564--red-hat-design-system.netlify.app/elements/accordion/style/#accent-slot) 

## Notes to Reviewers
Closes #1539
